lib/web/apiserver.go

```diff
@@ -2048,7 +2045,8 @@ type getSiteNamespacesResponse struct {
 	Namespaces []types.Namespace `json:"namespaces"`
 }
 
-/* getSiteNamespaces returns a list of namespaces for a given site
+/*
+	getSiteNamespaces returns a list of namespaces for a given site
```

Comment:
nitpick: feels weird this line uses a tab now and does not align with the rest. Not a big deal either way but maybe we can convert `/* */` to `//` like others.